src/mutations/item.ts

```diff
@@ -208,10 +218,17 @@ export default (queryConfig: QueryClientConfig) => {
           return previousItems;
         },
         onSuccess: () => {
-          notifier?.({
-            type: editItemRoutine.SUCCESS,
-            payload: { message: SUCCESS_MESSAGES.EDIT_ITEM },
-          });
+          if (
+            isNotificationEnabled(
+              enableNotifications,
+              NotificationStatus.SUCCESS,
+            )
+          ) {
+            notifier?.({
+              type: editItemRoutine.SUCCESS,
+              payload: { message: SUCCESS_MESSAGES.EDIT_ITEM },
+            });
```

Comment:
What about changing notifier to have a second parameter that would be `enableNotifications`/`options`? then the notifier that we pass can decide how to behave in those cases (eg. console)